storybook/qmlTests/tst_SessionRequestWithAuth.qml

```diff
@@ -0,0 +1,155 @@
+import QtQuick 2.15
+
+import QtTest 1.15
+
+import AppLayouts.Wallet.services.dapps.types 1.0
+
+import shared.stores 1.0
+
+Item {
+    id: root
+
+    width: 600
+    height: 400
+
+
+    Component {
```

Comment:
What about just nesting the store in `sessionRequestComponent` component, by binding directly to the `store` property?

Reply:
Sounds good to me 😄 Would simplify the init